WooCommerce/src/androidTest/kotlin/com/woocommerce/android/screenshots/orders/OrderCreationScreen.kt

```diff
@@ -14,11 +15,17 @@ class OrderCreationScreen : Screen {
 
     constructor() : super(ORDER_CREATION)
 
-    fun createEmptyOrder(): SingleOrderScreen {
+    fun createOrder(): SingleOrderScreen {
         clickOn(CREATE_BUTTON)
         return SingleOrderScreen()
     }
 
+    fun addProduct(): OrderSelectProductScreen {
```

Comment:
(Just thinking aloud, not sure if this is really needed.) The current name might give an impression that this method executes the complete process of product addition, while it only taps the `Add product` button. Maybe it makes sense to rename it to 🤔 

```suggestion
    fun addProductTap(): OrderSelectProductScreen {
```